libs/sdk-ui-kit/src/Header/HeaderHelp.tsx

```diff
@@ -77,13 +78,31 @@ export const CoreHeaderHelp: React.FC<IHeaderHelpProps> = ({
         onMenuItemClick(...args);
     };
 
+    /**
+     * Menu dropdown content is long enough to make it max-width (230px),
+     * so it should just switch alignment to bottom right corner of the Help button.
+     */
+    const getHelpDropdownAlignPoints = (): HelpMenuDropdownAlignPoints => {
+        const helpMenuCurrentRef = helpMenuRef?.current;
+        if (
+            !helpMenuCurrentRef ||
+            !helpMenuDropdownAlignPoints ||
+            helpMenuDropdownAlignPoints === "br tr" ||
+            !(window.innerWidth - helpMenuCurrentRef.offsetLeft >= 240)
```

Comment:
Should we rewrite as follows, to be somewhat more readable

```
!(window.innerWidth - helpMenuCurrentRef.offsetLeft >= 240)
```

 ↓

```
window.innerWidth - helpMenuCurrentRef.offsetLeft < 240
```

Reply:
Also, just a confirmation, should it be 230 rather than 240, as mentioned in https://github.com/gooddata/gooddata-ui-sdk/pull/2731#issue-1298640112?

![image](https://user-images.githubusercontent.com/67769750/179893554-5ae5a62b-be1d-433c-8d33-93fcf5125412.png)

Reply:
I remove the code. Please check again